README.md

```diff
@@ -627,7 +627,62 @@ This extension must be activated separately in the project conf.py
   res0: Int = 14
 ```
 
-### 13. Media directives
+### 13. Submittable ACOS exercises
+
+The custom directive acos-submit behaves almost identically to the normal
+submit directive. It is intended for exercises that are hosted outside the MOOC grader,
+such as the ACOS server. The directive option url should define the URL path of
+the exercise in the ACOS server. The URL domain is added automatically based on
+the configuration value `acos_submit_base_url` in conf.py. The acos-submit
+directive also automatically uses the `ajax` flag of the submit directive.
+
```

Comment:
Let's add a simple example of the `acos_submit_base_url` configuration setting and the `acos-submit` directive.

Something like this:

```
acos_submit_base_url = 'https://acos.cs.aalto.fi'
acos_submit_base_url = 'http://172.21.0.4:3000'

.. acos-submit:: somekey 100
  :title: My title
  :url: /aplus/draganddrop/draganddrop-example/revealdemo
```

Reply:
Since the example code block shows two different things now, add these headings there:

```
In conf.py:
acos_submit_base_url = 'https://acos.cs.aalto.fi'
acos_submit_base_url = 'http://172.21.0.4:3000'

In RST:
.. acos-submit:: somekey 100
  :title: My title
  :url: /aplus/draganddrop/draganddrop-example/revealdemo
```

Also add text "Usage example:" before the code block.